cluster/test_cluster.go

```diff
@@ -169,3 +170,34 @@ func getAddrFunc(seed int) func(*testing.T) *net.TCPAddr {
 		}
 	}
 }
+
+// MarshalPartialDefinition returns a json marshaller which marshals partial definitions, i.e., definitions with empty operators list
+// and empty definition-hash.
+func MarshalPartialDefinition(t *testing.T) func(def Definition) ([]byte, error) {
```

Comment:
this is only used another test package, suggest moving it there. Also why return a function, just marshal it and return the bytes.

Reply:
`definitionJSONv1x4` is private